### PR TITLE
API endpoints for settings

### PR DESCRIPTION
**Pull request by sajjanstha**
### Task/Ticket

[API endpoints for settings](#)

#### The main changes are

- [x] Create setting table
- [x] Endpoints for create and fetch settings

### CODE REVIEW

File: src/types/setting.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+
+export const SettingRequestSchema = z.object({
+  bannerImage: z.string().nullable(),
```
Comment (sazanrjb) [inline src/types/setting.ts:4]: these should be optional field in requests

**Comment by sazanrjb** [comment]
looks like both npm and yarn was used.. lets use one.. are you using yarn @aatbip?

**Comment by sazanrjb** [comment]
Build is failing. Looks like eslint issue. Please reformat and push https://vercel.com/outsidetech/copilot-client-home/4ZXESx1XzyTCKhu79F44W1ccmRgv

**Review by sazanrjb — APPROVED** [review]

**Comment by aatbip** [comment]
> looks like both npm and yarn was used.. lets use one.. are you using yarn @aatbip?

yes, I am using yarn.

**Pull request merged**